src/components/displayDocuments/renderQrCode.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import QRCode from "qrcode.react";
+
+interface RenderQrCodeProps {
+  url: string;
+}
+
+export const RenderQrCode: React.FunctionComponent<RenderQrCodeProps> = ({ url }) => {
```

Comment:
QrCode as name is enough ( or maybe you dont need that component ?)

Reply:
removed